pkg/awsutils/awsutils.go

```diff
@@ -78,6 +80,10 @@ const (
 
 	// the default page size when paginating the DescribeNetworkInterfaces call
 	describeENIPageSize = 1000
+
+	// enableImdsOnlyMode is used to specify that EC2 API calls should be skipped and only IMDS metadata should be
+	// used to get ENI information.
```

Comment:
You don't need to define the variable here again

Reply:
but we cannot import from ```ipamd``` as that will introduce circular imports. Didn't find a file for putting constant in current repo. I can have a new file for shared constant